sql/alps_submitter.go

```diff
@@ -4,12 +4,26 @@ import (
 	"bytes"
 	"fmt"
 	"io"
-	"io/ioutil"
 	"strconv"
 	"strings"
 	"text/template"
 )
 
+/*
+	FIXME(uuleon)
+
+	Attention:
+
+	In general, the `model_dir` should be a temporary directory for each `train` job who saves the output model to DB, then
+	the `predict` job restores model from DB.
+
+	But, the `maxCompute` type of DB which `ALPS` used is not supported yet, so a fixed `model_dir` is used here.
+
+	The `train` job clear the `model_dir` at first before training, then the `predict` job restores the output model of last `train` job.
+
+*/
+const modelDir = "/tmp/alps_model_dir/"
```

Comment:
Would multiple simultaneous Alps jobs share the same directory? If so, would they conflict with each other?

Reply:
You are right. That's why a `FIXME` is added here, and the confliction will be resolved after the DB of `MaxCompute` is available.